src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -1258,21 +1258,21 @@ private void DecodeInterpolatedStringHandlerArgumentAttribute(ref DecodeWellKnow
             TypedConstant constructorArgument = arguments.Attribute.CommonConstructorArguments[0];
 
             ImmutableArray<ParameterSymbol> containingSymbolParameters = ContainingSymbol.GetParameters();
+            // If we're analyzing the extension marker method itself, then we're not actually in an extension context, even though our containing type
+            // is the extension skeleton type.
```

Comment:
>extension skeleton type. [](http://example.com/codeflow?start=22&length=24)

We are not using "skeleton" term with extensions and it was never used to refer to an extension block type. Please rephrase.